getting_started/step_by_step/7_room_two.rst

```diff
@@ -205,15 +207,16 @@ Add the following code to room_pub.esc
 
 .. code-block::
 
-    > [eq ESC_LAST_SCENE room_park]
-       teleport player pub_exit
+    > [eq ESC_LAST_SCENE park]
+       teleport graham pub_exit
        # Set player look left
-       set_angle player 180
+       set_angle graham 180
 
 This code says
 1) Check (">") if the last room you were in (ESC_LAST_SCENE) is (eq) "room_park"
-2) If so, "teleport" the "player" to the ``ESClocation`` called "pub_exit"
-3) Then make the player face left
+2) If so, "teleport" the object "graham" to the ``ESClocation`` called
```

Comment:
```suggestion
2) If so, `teleport` the object "graham" to the ``ESCLocation`` called
```

Reply:
One day I'll get the quotes right... One day...